docs/get-started.md

```diff
@@ -209,6 +209,40 @@ Commands that follow this style are:
 
 For machines other than `default`, and commands other than those listed above, you must always specify the name explicitly as an argument.
 
+## Start local machines on startup
+
+In order to ensure that the Docker client is automatically configured at the start of each shell session, some users like to embed `eval $(docker-machine env default)` in their shell profiles (e.g., the `~/.bash_profile` file). However, this will fail if the `default` machine is not running. If desired, you can configure your system to start the `default` machine automatically.
```

Comment:
Nit; "this will fail" -> "this fails"?

Also, this should probably be wrapped to 80 chars (but I see the rest of this doc isn't wrapped, so just a nit)